src/lib/Tests/UI/Config/Mapper/FieldType/RichText/CustomTagTest.php

```diff
@@ -32,6 +32,7 @@ public function testMapConfig(array $customTagsConfiguration, array $enabledCust
         $mapper = new CustomTag(
             $customTagsConfiguration,
             $this->getTranslatorMock(),
+            $this->getTranslatorMock(),
```

Comment:
Is there any way that those could be split into `TranslatorInterface` mock and `TranslatorBagInterface`?

Reply:
Sure: https://github.com/ezsystems/ezplatform-admin-ui/pull/1335/commits/8c72a020245bf22ef482cb97aaa3bdabc65b211f